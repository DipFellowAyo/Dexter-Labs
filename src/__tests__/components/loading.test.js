import React from 'react'
import CircularProgress from '@material-ui/core/CircularProgress'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'

import Loading from '../../components/loading'

describe('Loading', () => {
  let loader

  beforeEach(() => {
    loader = shallow(<Loading />)
  })
  afterAll(() => jest.resetAllMocks())

  it('renders correctly', () => {
    expect(toJson(loader)).toMatchSnapshot()
  })

  it('CircularProgress accepts a props `color`', () => {
    expect(loader.find(CircularProgress).props().color).toBe('secondary')
  })
})
