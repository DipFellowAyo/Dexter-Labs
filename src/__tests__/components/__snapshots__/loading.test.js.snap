// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<Styled(MuiBox)
  className="makeStyles-centered-1"
>
  <WithStyles(ForwardRef(CircularProgress))
    color="secondary"
  />
</Styled(MuiBox)>
`;
